src/docs/admin/references/properties/scheduler.properties

```diff
@@ -10,24 +10,24 @@ pa.scheduler.home=.
 # you can use an absolute location or a relative location from scheduler home
 # pa.hsqldb.location=data/db
 
-# Scheduler REST url. If not defined, it is set automatically when starting the server.
+# Scheduler rest url. If not defined, it is set automatically when starting the server.
 # When the server has a public endpoint, different from the hostname available on the machine, this property should be used to correctly set the url
 # Accessed inside workflows and tasks with the PA_SCHEDULER_REST_URL variable
 #pa.scheduler.rest.url=
 
-# Catalog REST url. If not defined, it is set automatically when starting the server.
+# Catalog rest url. If not defined, it is set automatically when starting the server.
 # Accessed inside workflows and tasks with the PA_CATALOG_REST_URL variable
 #pa.catalog.rest.url=
 
-# Service Automation REST url. If not defined, it is set automatically when starting the server.
+# Cloud Automation rest url. If not defined, it is set automatically when starting the server.
```

Comment:
ok for me if you prefer to use old naming on the server side. Which is normal as we didn't change microservice name

Reply:
same answer